setup.sh

```
@@ -22,13 +22,40 @@ elif ! node -v | grep -q "^v20\\." ; then
+        corepack enable pnpm
+      fi
+    else
+      echo "▶ No valid packageManager field found, using default pnpm"
+      corepack enable pnpm
+    fi
+  else
+    # Fallback when jq is not available or package.json is missing
+    echo "▶ jq not available or package.json missing, using default pnpm"
+    corepack enable pnpm
+  fi
 fi
```

Shellcheck (0.10.0)

[error] Parsing stopped here. Is this keyword correctly matched up?

(SC1089)